app/models/miq_worker/runner.rb

```diff
@@ -92,7 +92,7 @@ def worker_monitor_drb
   end
 
   def start
-    self.class.module_parent.rails_worker? ? start_rails_worker : start_non_rails_worker
+    self.class.module_parent.rails_worker ? start_rails_worker : start_non_rails_worker
```

Comment:
I don't understand this change.  I would expect to run the `?` method which actually resolves the proc, as opposed to the presence of a proc.

Reply:
This was an old branch I just rebased, this change isn't relevent anymore